sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,16 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self.args[0].is_real:
+            return True
```

Comment:
Yes, the checks are made by taking those forms of the expression as @jksuom said.